build/ci/release.yml

```diff
@@ -612,17 +646,6 @@ tasks:
           bucket: mongodb-mongocli-build
           permissions: public-read
           content_type: ${content_type|text/plain}
-      - command: s3.put
```

Comment:
is this intentional?

Reply:
it is, msis are only uploaded after signing after goreleaser happens